README.md

```diff
@@ -37,7 +45,14 @@ yarn add @unstoppabledomains/resolution
 npm install @unstoppabledomains/resolution --save
 ```
 
-If you're interested in resolving domains via the command line, see our [CLI section](#command-line-interface). 
+If you're interested in resolving domains via the command line, see our
+[CLI section](#command-line-interface).
+
+## Obtaining JSON RPC Provider
+
+Resolution for .crypto and .eth performs reads on the Ethereum blockchain and
+requires a link to an Ethereum JSON RPC provider. You can obtain a provider url
+by signing up with [Infura](https://infura.io/)
```

Comment:
@sudoryan It's worth to mention other providers as well and provide a link to the article about Ethereum providers.

https://ethereum.org/en/developers/docs/nodes-and-clients/nodes-as-a-service/